CRM/Membershipextrasdata/Utils.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class CRM_Membershipextrasdata_Utils {
```

Comment:
I think this class should have it own class name under utils package (name space). For example `CRM_Membershipextrasdata_Utils_xxx`

Reply:
I created `CRM_Membershipextrasdata_Utils_String`